pkgs/by-name/sc/scooter/package.nix

```
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "scooter";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "thomasschafer";
+    repo = "scooter";
+    rev = "v${version}";
+    hash = "sha256-5YosLJHfrXv6ev7r0gAW5bzkSIwIBUj/Fl3+tklTKQo=";
+  };
+
+  cargoHash = "sha256-GY52JN07igDJVuKXR9iIqNj1obhPs70dOmU1wGy8ZrI";
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
See https://github.com/NixOS/nixpkgs/issues/208242 https://github.com/NixOS/nixpkgs/pull/293767